src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/utils/DueDiligenceOptionValidators.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation.utils;
+
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.DueDiligenceDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntityDueDiligenceDto;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.OverseasEntitySubmissionDto;
+import uk.gov.companieshouse.overseasentitiesapi.utils.ApiLogger;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+import java.util.Objects;
+
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.UtilsValidators.setErrorMsgToLocation;
+import static uk.gov.companieshouse.overseasentitiesapi.validation.utils.ValidationMessages.SHOULD_NOT_BOTH_BE_POPULATED_ERROR_MESSAGE;
+
+public class DueDiligenceOptionValidators {
+
+     public static boolean bothDueDiligenceOptionsNotSupplied(DueDiligenceDto dueDiligenceDto,
```

Comment:
Probably we should change the name, what do you think :)

Reply:
Yes, have added a suggestion for the method in the class above.